README.md

```diff
@@ -31,6 +31,15 @@ npm i -D gulp-cache
 yarn add -D gulp-cache
 ```
 
+## Arguments
+```js
+/**
+* @param gulp_action the action to run, if not found in cache
+* @param options see section bellow for posible options.
```

Comment:
`see section below for possible options.`

Reply:
Pasted your suggestion and committed it.